Frameworks/CoreFoundation/String.subproj/CFString.c

```diff
@@ -1192,6 +1192,11 @@ CFHashCode __CFStringHash(CFTypeRef cf) {
     }
 }
 
+// WINOBJC: _CFStringHash is for Foundation's benefit. It should not be consumed by anyone else.
+CF_PRIVATE CF_EXPORT CFHashCode _CFStringHash(CFStringRef str) {
```

Comment:
nit: but does that mean it's the right place?

---
In reply to: [109267433](https://github.com/Microsoft/WinObjC/pull/2399#discussion_r109267433) [](ancestors = 109267433)